README.md

```diff
@@ -84,552 +84,284 @@ We conducted a comprehensive evaluation of Qwen2.5-Omni, which demonstrates stro
+| RealWorldQA<sub>avg</sub>      | 70.3            | **71.9**   | 68.5          | -           | 
+| MME-RealWorld<sub>en</sub>     | **61.6**        | -          | 57.4          | -           | 
+| MM-MT-Bench                    | 6.0             | -          | **6.3**       | -           | 
+| AI2D                           | 83.2            | **85.8**   | 83.9          | -           | 
+| TextVQA<sub>val</sub>          | 84.4            | 83.2       | **84.9**      | -           | 
+| DocVQA<sub>test</sub>          | 95.2            | 93.5       | **95.7**      | -           | 
+| ChartQA<sub>test Avg</sub>     | 85.3            | 84.9       | **87.3**      | -           | 
+| OCRBench_V2<sub>en</sub>       | **57.8**        | -          | 56.3          | -           | 
 
 | Dataset                  | Qwen2.5-Omni-7B | Qwen2.5-VL-7B | Grounding DINO | Gemini 1.5 Pro | 
-|--------------------------|--------------|---------------|----------------|----------------|
-| Refcoco<sub>val</sub>    | 90.5         | 90.0          | **90.6**       | 73.2           | 
-| Refcoco<sub>textA</sub>  | **93.5**     | 92.5          | 93.2           | 72.9           | 
-| Refcoco<sub>textB</sub>  | 86.6         | 85.4          | **88.2**       | 74.6           | 
-| Refcoco+<sub>val</sub>   | 85.4         | 84.2          | **88.2**       | 62.5           | 
-| Refcoco+<sub>textA</sub> | **91.0**     | 89.1          | 89.0           | 63.9           | 
-| Refcoco+<sub>textB</sub> | **79.3**     | 76.9          | 75.9           | 65.0           | 
-| Refcocog+<sub>val</sub>  | **87.4**     | 87.2          | 86.1           | 75.2           | 
-| Refcocog+<sub>test</sub> | **87.9**     | 87.2          | 87.0           | 76.2           | 
-| ODinW                    | 42.4         | 37.3          | **55.0**       | 36.7           | 
-| PointGrounding           | 66.5         | **67.3**      | -              | -              | 
-</details>
+|--------------------------|-----------------|---------------|----------------|----------------|
+| Refcoco<sub>val</sub>    | 90.5            | 90.0          | **90.6**       | 73.2           | 
+| Refcoco<sub>textA</sub>  | **93.5**        | 92.5          | 93.2           | 72.9           | 
+| Refcoco<sub>textB</sub>  | 86.6            | 85.4          | **88.2**       | 74.6           | 
+| Refcoco+<sub>val</sub>   | 85.4            | 84.2          | **88.2**       | 62.5           | 
+| Refcoco+<sub>textA</sub> | **91.0**        | 89.1          | 89.0           | 63.9           | 
+| Refcoco+<sub>textB</sub> | **79.3**        | 76.9          | 75.9           | 65.0           | 
+| Refcocog+<sub>val</sub>  | **87.4**        | 87.2          | 86.1           | 75.2           | 
+| Refcocog+<sub>test</sub> | **87.9**        | 87.2          | 87.0           | 76.2           | 
+| ODinW                    | 42.4            | 37.3          | **55.0**       | 36.7           | 
```

Comment:
```suggestion
| ODinW                    | 42.2            | 37.3          | **55.0**       | 36.7           | 
```

This is a typo from original markdown. Change it? 
<img width="533" alt="Screenshot 2025-04-08 at 23 44 41" src="https://github.com/user-attachments/assets/21059e85-e743-4d03-aa89-bc0ce45f9101" />